Jenkinsfile

```diff
@@ -7,20 +7,41 @@ pipeline {
         stage('Checkout'){
             steps {
                 sh """
-                    git clone https://github.com/couchbaselabs/${env.PRODUCT}.git
-                    pushd ${env.PRODUCT}
+                    #!/bin/bash
+                    set -e
+                    shopt -s extglob dotglob
+
+		    # move PR related repo to tmp folder
+		    mkdir tmp
+                    mv !(tmp) tmp
+                    git clone https://github.com/couchbaselabs/${env.PRODUCT}.git --branch $CHANGE_TARGET
```

Comment:
I think moving this line under # restructure folders to be together with `mv couchbase-lite-ios-ee/* ` feel less confusing.

Reply:
env.PRODUCT is only used once here so I don't think we need to declare it as a PRODUCT environment.